src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -1,99 +1,71 @@
-        setIsNoUser(true);
-      } else {
-        close();
-        setIsNoUser(false);
-      }
+      dispatch(addInstructor(selectedInstitution.id, formJson.instructorEmail));
+      close();
     } catch (error) {
       logError(error);
     }
   };
 
-  useEffect(() => {
-    dispatch(fetchClassesData());
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
-
   return (
-    <>
-      <Button variant="outline-primary" size="sm">
-        Add Instructor
-      </Button>
-      <ModalDialog
-        title="Add Instructor"
-        isOpen={isOpen}
-        onClose={close}
-        hasCloseButton
-      >
-        <ModalDialog.Header>
-          <ModalDialog.Title>
-            Add Instructor
-          </ModalDialog.Title>
-        </ModalDialog.Header>
-        <ModalDialog.Body>
-          <Form onSubmit={handleAddInstructors}>
-            <FormGroup controlId="ccxId">
-              <Form.Control
-                as="select"
-                floatingLabel="Select Class Name"
-                className="my-4 mr-0"
-                name="ccxId"
-              >
-                <option disabled value="null">Select an Option</option>
-                {stateInstructors.data.map((ccx) => <option value={ccx.classId}>{ccx.className}</option>)}
-              </Form.Control>
-            </FormGroup>
-            <FormGroup controlId="instructorInfo">
-              <Form.Control
-                type="text"
-                placeholder="Enter Username or Email of the instructor"
-                floatingLabel="Username or Email"
-                className="my-4 mr-0"
-                name="instructorInfo"
-              />
-              {isNoUser && (
-                <Form.Control.Feedback type="invalid">
-                  User does not exist
-                </Form.Control.Feedback>
-              )}
-            </FormGroup>
-            <div className="d-flex justify-content-end">
-              <Button type="submit">Add</Button>
-            </div>
-          </Form>
-
-        </ModalDialog.Body>
-      </ModalDialog>
-    </>
+    <ModalDialog
+      title="Add Instructor"
+      isOpen={isOpen}
+      onClose={close}
+      hasCloseButton
+    >
+      <ModalDialog.Header>
+        <ModalDialog.Title>
+          Add new instructor
+        </ModalDialog.Title>
+      </ModalDialog.Header>
+      <ModalDialog.Body>
+        <Form onSubmit={handleAddInstructor}>
+          <FormGroup controlId="instructorInfo">
+            <Form.Control
+              type="email"
+              placeholder="Enter Email of the instructor"
+              floatingLabel="Email"
+              className="my-4 mr-0"
+              name="instructorEmail"
+              required
+            />
+          </FormGroup>
+          <div className="d-flex justify-content-end">
+            <ModalCloseButton className="btntpz btn-text btn-tertiary">Cancel</ModalCloseButton>
+            <Button type="submit">Send invite</Button>
```

Comment:
```suggestion
            <Button type="submit">Send invitation</Button>
```

Reply:
This is how is in the figma, should I change it?